include/alp/reference/blas1.hpp

```diff
@@ -2634,15 +2639,15 @@ namespace alp {
 		void >::type * const = NULL
```

Comment:
Commenting here since I cannot do this near the signature above.
Are we using this variant variant of `dot` anyware? Shouldn't the one with `alp::Scalar` IO be sufficient?

Reply:
We don't use it, it was already in the sparse backed.